gramps/gen/db/base.py

```diff
@@ -25,21 +25,21 @@
 from this class.
 """
 
-#-------------------------------------------------------------------------
+# -------------------------------------------------------------------------
```

Comment:
Interesting.  I don't get a warning.  We use these standard comments everywhere.